src/components/table/table.stories.js

```diff
@@ -88,22 +112,20 @@ const buildRows = (pageSize, totalRecords) => {
   );
 };
 
+const setCurrentPage = ({ totalRecords, pageSize }) => {
+  if (totalRecords % pageSize !== 0) return store.get('currentPage');
+  return (store.get('currentPage') > (totalRecords / pageSize)) ? '1' : store.get('currentPage');
```

Comment:
could you extract the conditional to a separate `const`, to increase readability?

Reply:
Yeah good point, I had to refactor it slightly as I had missed an edge case